### PR TITLE
Fix scale control default parameters

### PR DESCRIPTION
**Pull request by vicb**
Before this CL, the following would generate an error:

```ts
const scaleCtrl = new ScaleControl():
```

because options is not marked as optional.

## Launch Checklist

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [ ] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [x] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

**Review by HarelM — APPROVED** [review]

**Comment by HarelM** [comment]
Can you please add a changelog item?

**Comment by vicb** [comment]
@HarelM thanks for the quick feedback. Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 ### 🐞 Bug fixes
 - Fix different unwanted panning changes at the end of a panning motion, that happen on a large screen ([#3935](https://github.com/maplibre/maplibre-gl-js/issues/3935))
 - Fix image sources not being marked as loaded on error
+- Fix ScaleControl options should be optional
```
[the remark this answers is not in the document]
Comment (vicb) [reply CHANGELOG.md:9]: I only see a link to an issue on the first item and no link on the second item.
But I'm happy to add it if I should.
Let me know!

[the remark this answers is not in the document]
Comment (HarelM) [reply CHANGELOG.md:9]: Either a pr link or issue, yes, I probably approve something I shouldn't have...

**Comment by vicb** [comment]
Done.
It would be nice to switch to something like conventional change logs to auto generate the change log

**Comment by HarelM** [comment]
Feel free to push this auto changelog forward, I didn't find something that is easy enough to use and is clear how the commit messages should look in order for it to work properly...
But I might haven't searched enough.

**Review by HarelM — APPROVED** [review]

**Pull request merged**